src/components/TextInput.tsx

```diff
@@ -26,17 +26,28 @@ const Asterix = styled(Label)`
 `;
 
 const Input = styled.input<{ error: boolean }>`
-	border-radius: 4px;
-	font-size: 0.8em;
 	border: none;
-	width: 100%;
+	border: 1px solid #bdbdbd;
+	border-radius: 10px;
+	outline: none;
+	// outline: auto 1px #bdbdbd;
+	font-size: 0.8em;
+	color: black;
+	width: 10em;
+	height: 2em;
 	padding: 0.25em;
+	&:focus {
+		border: 1px solid #aaaaaa;
+	}
+	&::placeholder {
+		color: #bdbdbd;
+	}
 	${(props) =>
 		props.error &&
 		css`
-			outline: auto 4px red; // Need to pick an error colour?
+			border: 2px solid #ff8888; // Need to pick an error colour?
```

Comment:
Yeah we need to create our custom theme. This has yet to be done, and the person responsible will look for hardcoded colours and change this. I think this is a nice colour though.

Reply:
I don't even remember changing this haha but for sure